src/test/resources/mapping/given_inventory_instance.json

```diff
@@ -0,0 +1,3 @@
+{
+  "title": "The Journal of ecclesiastical history"
```

Comment:
Once we have more mapping rules, we will expand this instance with more new fields, that are affected by mapping